### ISSUE TITLE
Password entry for SSH connection should have keyboard focus

### ISSUE THREAD

**Issue by aruiz** [fleet-commander/fc-admin#102]

**Comment by olivergs** [comment]
Good catch. This and Issue #103 are things we should standarize application wide. In Hypervisor dialog we should focus first field etc.
